lib/migration/plugins/plugin9/container_create_timestamp_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plugin9
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/vmware/vic/pkg/vsphere/extraconfig"
+)
+
+// TestMigrateContainerCreateTimestamp tests this package's ContainerCreateTimestampVersion.Migrate.
+func TestMigrateContainerCreateTimestamp(t *testing.T) {
+	secSinceEpoch := int64(1510950922)
+	oldCreateTime := secSinceEpoch
+	newCreateTime := secSinceEpoch * 1e9
+	execConfig := ExecutorConfig{
```

Comment:
I recommend having another field in here as well as the one being migrated - that ensures that we're not accidentally dropping data that doesn't need migration and is ignored by the specialized struct.